FEXCore/Source/Interface/Core/Core.cpp

```diff
@@ -419,6 +412,13 @@ namespace FEXCore::Context {
     Thread->CurrentFrame->State.DeferredSignalRefCount.Store(0);
     Thread->CurrentFrame->State.DeferredSignalFaultAddress = reinterpret_cast<Core::NonAtomicRefCounter<uint64_t>*>(FEXCore::Allocator::VirtualAlloc(4096));
 
+    if (Config.BlockJITNaming() ||
+        Config.GlobalJITNaming() ||
+        Config.LibraryJITNaming()) {
+      // Allocate a TLS JIT symbol buffer only if enabled.
```

Comment:
if it's not TLS, does the comment need adjustment?

Reply:
Updated comment